manager/state/raft/raft.go

```diff
@@ -914,7 +936,7 @@ func (n *Node) configure(ctx context.Context, cc raftpb.ConfChange) error {
 	case <-ctx.Done():
 		n.wait.trigger(cc.ID, nil)
 		return ctx.Err()
-	case <-n.stopCh:
+	case <-n.Ctx.Done():
```

Comment:
@abronan This code needs further refactoring to remove the `Ctx` field from node. This wasn't introduced here, waiting on context is the right thing to do.